package.json

```diff
@@ -2,14 +2,20 @@
   "name": "renovate-config",
   "version": "0.0.0-development",
   "description": "Renovate config presets for some Capra projects.",
+  "scripts": {
+    "prepare": "husky install"
+  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/capraconsulting/renovate-config.git"
   },
   "author": "Capra Consulting",
   "license": "Apache 2.0",
   "devDependencies": {
+    "@commitlint/cli": "^18.4.4",
```

Comment:
Minor thing in this repository, especially as its just dev dependencies, but we generally prefer to pin all dependencies to exact versions

Reply:
Yeah, I don't really have any strong opinions on this, I'll pin it

Reply:
https://github.com/capralifecycle/renovate-config/pull/41/commits/4c1f8c582fd5f6d2804ab54bf1948c866f3b080d